vagrant/src/main/vagrant/files/install_brooklyn.sh

```diff
@@ -78,12 +78,12 @@ tar zxf ${TMP_ARCHIVE_NAME}
 echo "Creating Brooklyn dirs and symlinks"
 ln -s ${BROOKLYN_DIR} apache-brooklyn
 sudo mkdir -p /var/log/brooklyn
-sudo chown -R vagrant:vagrant /var/log/brooklyn
-mkdir -p /home/vagrant/.brooklyn
+sudo chown -R ubuntu:ubuntu /var/log/brooklyn
+mkdir -p /home/ubuntu/.brooklyn
```

Comment:
For some reason the `ubuntu/xenial64` image used ubuntu as the default user, I didn't look into it further.